src/main/java/com/documaster/rms/noark/ws/noarkentities/expand/UtvidTilJournalpost.java

```diff
@@ -0,0 +1,166 @@
+package com.documaster.rms.noark.ws.noarkentities.expand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.documaster.rms.noark.ws.client.Bean;
+import com.documaster.rms.noark.ws.constants.Journalposttype;
+import com.documaster.rms.noark.ws.constants.Journalstatus;
+import com.documaster.rms.noark.ws.constants.Korrespondanseparttype;
+import com.documaster.rms.noark.ws.serialization.NoarkEnumJsonSerializer;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+public class UtvidTilJournalpost implements Bean {
+
+	private String id;
+	private Journalpost journalpost;
+	private List<Korrespondansepart> korrespondansepart = new ArrayList<>();
+
+	public String getId() {
+
+		return id;
+	}
+
+	public void setId(String id) {
+
+		this.id = id;
+	}
+
+	public Journalpost getJournalpost() {
+
+		return journalpost;
+	}
+
+	public void setJournalpost(Journalpost journalpost) {
+
+		this.journalpost = journalpost;
+	}
+
+	public List<Korrespondansepart> getKorrespondansepart() {
+
+		return korrespondansepart;
+	}
+
+	public void setKorrespondansepart(
+			List<Korrespondansepart> korrespondansepart) {
+
+		this.korrespondansepart = korrespondansepart;
+	}
+
+	public static class Journalpost {
```

Comment:
Should we also include `registreringsIdent` and `registreringsDato`?

Reply:
I didn't on purpose, because they will be auto-generated with very reasonable defaults. Integrations can always send an update request to modify them later.